arch/xtensa/src/esp32/Kconfig

```diff
@@ -1891,6 +1891,12 @@ config ESP32_STORAGE_MTD_SIZE
 	---help---
 		MTD size in SPI Flash.
 
+mtd_name = ESP32_STORAGE_MTD
+mtd_part_default_sizes = 8,504,512
+mtd_part_default_names = mtd0p0,mtd0p1,mtd0p2
```

Comment:
I think this is the first time I see this kind of usage inside Kconfig. Maybe there are more standard way to implement the same thing. @simbit18 and @xiaoxiang781216 please comment.